client/components/AppHeader.tsx

```diff
@@ -0,0 +1,129 @@
+                            <Typography
+                                variant='h6'
+                            >
+                                Please Log In
+                            </Typography>
+                        )}
+                    </IconButton>
+                    <Menu
+                        id="menu-account"
+                        anchorEl={accountCircleAnchorEl}
+                        anchorOrigin={{
+                        vertical: 'top',
+                        horizontal: 'right',
+                        }}
+                        keepMounted
+                        transformOrigin={{
+                        vertical: 'top',
+                        horizontal: 'right',
+                        }}
+                        open={Boolean(accountCircleAnchorEl)}
+                        onClose={e => setAccountCircleAnchorEl(null)}
+                    >
+                        <MenuItem 
+                            onClick={e => {
+                                setAccountCircleAnchorEl(null);
+                                navigate("/me");
+                            }}
+                        > 
+                            Profile
+                        </MenuItem>
+                        <MenuItem
+                            onClick={e => {
+                                setAccountCircleAnchorEl(null);
+                                dispatch(logout());
+                                navigate("/");
+                            }}
+                        >
+                            Logout
+                        </MenuItem>
+                    </Menu>
+                </Box>
+
+                <NavLink to="/" >
+                    <img src="/images/Zoop_logo_withtext.svg"/>
+                </NavLink>
+
+                
+                <Box component='div'>
+                    {!currentUser && (
+                        <ButtonGroup
+                            variant='contained'
+                            aria-label='contained button group'
+                        >
+                            <Button 
+                                color='primary'
+                                onClick={() => {navigate('/login')}}
+                            >
+                                Login
+                            </Button>
+                            <Button 
+                                color='secondary'
+                                onClick={() => {navigate('/register')}}
+                            >
+                                Sign Up
+                            </Button>
+                        </ButtonGroup>
+                    )}
+                </Box>
+            </Stack>
+            {/* TODO: move this to App.tsx */}
+            <NavLink to="/">Home</NavLink>
+            <NavLink to="/login">Login</NavLink>
+            <NavLink to="/register">Register</NavLink>
+            <NavLink to="/me">Me</NavLink>
```

Comment:
do you still need this? couldn't it be deleted now that you have the navbar?

Reply:
Indeed this was just here for dev purposes before the navbar was done. I'll remove it.